frontend/src/components/Listing/ListingCard.tsx

```diff
@@ -46,6 +83,28 @@ export default function ListingCard (props: IListingCardProps) {
           </Typography>
           <AddShoppingCartIcon className='icon-spacing'/>
         </Button>
+          <Button
+              variant="contained"
+              color="primary"
```

Comment:
![image](https://user-images.githubusercontent.com/72239525/226495531-7b72054c-1284-40a4-b532-3bf2044f9ef9.png)
Hide this button when not logged in.
Add ` const { isLoggedIn } = useContext(UserLoginContext);`
to this function and use it as a bool that shows/hides the edit button

Reply:
Added functionality described in the comment